blockchain/chain_sync/src/sync.rs

```diff
@@ -729,6 +736,99 @@ where
         Ok(())
     }
 
+    pub async fn verify_winning_posts_proof(
+        &self,
+        block: BlockHeader,
+        prev_entry: BeaconEntry,
+        lbst: Cid,
+    ) -> Result<(), Error> {
+        if block.win_post_proof().is_empty() {
+            return Err(Error::Validation(
+                "[TESTING] No winning post proof given".to_string(),
+            ));
+        }
+
+        if let Some(first_proof) = block.win_post_proof().first() {
+            if from_utf8(&first_proof.proof_bytes)
+                .map(|buf| buf == "valid proof")
+                .unwrap_or_default()
+            {
+                return Ok(());
+            }
+        }
```

Comment:
They only use this when an insecure validation flag is set for testing. We don't have this option so should just leave out for now